x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/WaitForIndexColorStepTests.java

```diff
@@ -54,18 +54,10 @@ protected WaitForIndexColorStep mutateInstance(WaitForIndexColorStep instance) {
         }
 
         switch (between(0, 2)) {
-            case 0:
-                key = new StepKey(key.getPhase(), key.getAction(), key.getName() + randomAlphaOfLength(5));
-                break;
-            case 1:
-                nextKey = new StepKey(key.getPhase(), key.getAction(), key.getName() + randomAlphaOfLength(5));
-                break;
-            case 2:
-                color = newColor;
-                break;
-            case 3:
-                indexPrefix = randomAlphaOfLengthBetween(1, 10);
-                break;
+            case 0 -> key = new StepKey(key.getPhase(), key.getAction(), key.getName() + randomAlphaOfLength(5));
+            case 1 -> nextKey = new StepKey(key.getPhase(), key.getAction(), key.getName() + randomAlphaOfLength(5));
+            case 2 -> color = newColor;
+            case 3 -> indexPrefix = randomAlphaOfLengthBetween(1, 10);
```

Comment:
...but the values are only be between 0 and 2...?

Reply:
Let me remove the 3rd case.